### DISCUSSION TITLE
Why JS onclick event doesn't work with <a> links?

### DISCUSSION THREAD

**Discussion by marcela-fernanda** [requarks/wiki#3005] [Help / Questions]
I was able to use onclick with buttons, but links seems to not work

**Comment by NGPixel** [comment]
Do you have an example of what you're trying to achieve?

**Reply by marcela-fernanda** [reply]
I'm trying to use onclick to get when a link was clicked and, based on that, add a class to an element and remove from another. Basically, I'm trying to do this because I want to change the item that's open on the `<details>` section, like this:

![Payment_Slip_Invoice - Esquema de Dados - BAAS _ Documentação - Google Chrome 2021-02-05 08-10-34 (1)](https://user-images.githubusercontent.com/42184254/107026838-10cdc280-678a-11eb-95e7-50169b1d6a01.gif)

**Comment by NGPixel** [comment]
You can try to disable **Sanitization** under Admin > Rendering > HTML > Security. Then save your page again.

**Reply by marcela-fernanda** [reply]
I try it but didn't work
